tests/actions/test_action_typedicts.py

```
@@ -0,0 +1,234 @@
+
+        if typeddict_cls is not None:
+            mapping[action_type] = (action_cls, typeddict_cls)
+
+    return mapping
+
+
+def test_all_actions_have_typeddict() -> None:
+    """Test that every action in ACTION_REGISTRY has a corresponding TypedDict."""
+    missing_typedicts: list[str] = []
+
+    for action_type, action_cls in BaseAction.ACTION_REGISTRY.items():
```

Ruff (0.14.8)

Loop control variable `action_type` not used within loop body

Rename unused `action_type` to `_action_type`

(B007)